### DISCUSSION TITLE
Help me! What to choose for web development?

### DISCUSSION THREAD

**Discussion by neptship** [community/community#21025] [Codespaces] [answered]
VSC or WebStorm?

**Comment by DenverCoder1** [comment]
VS Code is a modern, lightweight editor that is very extendable and frequently developed for.

WebStorm is geared toward a very specific use case, so can have more powerful tools and more built-in features, but also is a bit slower and comes with many features you'll never need.

I personally would use VS Code over WebStorm for any project, especially since I work in many languages and VS Code has tools available for nearly everything.

With the amount of extensions available for VS Code, there isn't really much you can do in WebStorm that you can't do in VS Code, but there are a lot of things available for VS Code as extensions that you can't do easily in WebStorm.

**Comment by venus01015** [comment, marked as answer]
Visual Studio Code, also commonly referred to as VS Code, is a source-code editor made by Microsoft for Windows, Linux and macOS. Features include support for debugging, syntax highlighting, intelligent code completion, snippets, code refactoring, and embedded Git.
So VS Code is better than WebStorm.

**Reply by DenverCoder1** [reply]
All the features you mentioned are features in WebStorm as well as far as I know.
